api/src/pcapi/routes/backoffice/templates/bank_account/get.html

```diff
@@ -93,6 +93,12 @@ <h5 class="modal-title">Modifier les informations du compte bancaire</h5>
                     <span class="fw-bold">Statut DMS CB :</span>
                     {{ dms_stats.status | format_dms_status }}
                   </p>
+                {% endif %}
+                <p class="mb-1">
+                  <span class="fw-bold">Statut Bank</span>
```

Comment:
Sois attentif...
```suggestion
                  <span class="fw-bold">Statut du compte bancaire :</span>
```
Même si je reste personnellement partisant de « État » et non « Statut » comme traduction de « status ».